0-bootstrap/README.md

```diff
@@ -108,27 +108,21 @@ your current Jenkins manager (master) environment.
 ## Deploying with Cloud Build
 
 1. Go to the `0-bootstrap` folder.
-1. Copy the `tfvars` file:
-   ```
-   cp terraform.example.tfvars terraform.tfvars
-   ```
-1. Update the `terraform.tfvars` file with values from your environment.
+1. Rename `terraform.example.tfvars` to `terraform.tfvars` and update the file with values from your environment.
 1. Run `terraform init`.
 1. Run `terraform plan` and review the output.
 1. To run terraform-validator steps please follow the [instructions](https://github.com/forseti-security/policy-library/blob/master/docs/user_guide.md#install-terraform-validator) in the **Install Terraform Validator** section and install version `2021-03-22`. You will also need to rename the binary from `terraform-validator-<your-platform>` to `terraform-validator`.
     1. Run `terraform plan -input=false -out bootstrap.tfplan`
     1. Run `terraform show -json bootstrap.tfplan > bootstrap.json`
     1. Run `terraform-validator validate bootstrap.json --policy-path="../policy-library" --project <A-VALID-PROJECT-ID>` and check for violations (`<A-VALID-PROJECT-ID>` must be an existing project you have access to, this is necessary because Terraform-validator needs to link resources to a valid Google Cloud Platform project).
 1. Run `terraform apply`.
-1. Run `terraform output gcs_bucket_tfstate` to get your Google Cloud bucket
-   from the previous step.
+1. Run `terraform output terraform_service_account` to get the email address of the admin. You need this address in a later procedure.
```

Comment:
Is there a reason the ordering for this step has changed? Is it to group the `terraform output` steps?

Reply:
After updating the backend Terraform must  be initialized before other operations can be performed

Step 12 `terraform init` is required before step 11 `terraform output terraform_service_account`. The current process causes user to be prompted to run init when attempting step 11.

PR changes order to reflect the required order.